addons/l10n_be/models/res_partner.py

```diff
@@ -3,17 +3,17 @@
 
 # Copyright (c) 2011 Noviat nv/sa (www.noviat.be). All rights reserved.
 
-from odoo import api, models
+from odoo import api, models, fields
 
 
 class ResPartner(models.Model):
     _inherit = 'res.partner'
 
+    company_registry = fields.Char(compute='_compute_company_registry', store=True, readonly=False)
```

Comment:
This is a field needed in more and more l10n, and is already duplicated with many different names
It needs to be in `base` (or `account` at the very least)

Reply:
And making it a computed field only in one child module is a very bad idea IMO because it breaks the heritability

Reply:
Fair enough, I dropped the commit.